syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -73,6 +73,15 @@ def __init__(
                     "Unsupported arg value for dtype. Use dtype='long' or dtype='int'."
                 )
 
+    def __mod__(self, other):
+        """
+        Define the modulo operation over object instances.
+        """
+        result = self.copy()
```

Comment:
is there a reason why you decided to copy the object and not modify the self.child directly and return self?

Reply:
For inmutability of `x` in `x % y`. Would you say it's best to return `x`?

Reply:
no, it would be better to init a new instance :)